db/index.js

```diff
@@ -1,113 +1,80 @@
-const mysql = require('mysql');
-const config = require('./config.js');
+const Pool = require('pg').Pool
 
-const connection = mysql.createConnection(config);
-
-connection.connect((err) => {
-  if (err) {
-    console.log(err, 'ERROR CONNECTING');
-  } else {
-    console.log('connected');
-  }
+const pool = new Pool({
+  user: 'daniel',
+  host: 'localhost',
+  database: 'bookshelf',
```

Comment:
you may want to consider extracting your db credentials into a separate gitignored config file or as env variables

Reply:
most definitely. 
done!